panel/src/components/Navigation/DropdownContent.vue

```diff
@@ -29,16 +34,25 @@ export default {
   props: {
     options: [Array, Function],
     /**
-     * Aligment of the dropdown items
+     * Alignment of the dropdown items
      * @values left, right
      */
     align: {
       type: String,
       default: "left"
+    },
+    /**
+     * Vertical alignment of the dropdown menu
+     * @values true, false
+     */
+    dropup: {
```

Comment:
Does a prop make a lot of sense? Since the component is now "magically" taking care of this anyways, I don't think we need a prop to set the behaviour from outside the component, do we?

Reply:
I thought users might want to enable and use dropup by default.  Something like a footer menu.  I think it's something extreme 🙂

Reply:
Maybe it should then be possible to have `true|false|null` to be able to explicitly disable the dropup as well. `null` would be the default (for auto).

Reply:
I've removed the excess prop.

Reply:
> Maybe it should then be possible to have true|false|null to be able to explicitly disable the dropup as well. null would be the default (for auto).

@lukasbestle Sorry, I saw your comment now. Should I revert it?

Reply:
I'm fine with either implementation, but I agree with you that the prop to control the behavior could be useful. @distantnative?

Reply:
I think it's better to keep this as a data key and not reintroduce a prop. Aligning the dropdown left right or center makes sense as a manual decision. But I agree with @distantnative that the vertical direction should be handled automatically.